src/scenes/Species/SpeciesListView.tsx

```diff
@@ -312,29 +305,20 @@ export default function SpeciesListView({ reloadData, species }: SpeciesListProp
             },
           ],
         },
-        sortOrder: [{ field: 'id', direction: 'Ascending' }],
         count: 1000,
       };
 
       setIsBusy(true);
-      const data = await SearchService.search(searchParams);
+      const data = await SearchService.searchValues(searchParams);
       setIsBusy(false);
-      const result = (data ?? []).reduce((acc, d) => {
-        return Object.keys(d).reduce((innerAcc, k) => {
-          const isEcosystemTypes = k === 'ecosystemTypes';
-          const newKey = isEcosystemTypes ? 'ecosystemTypes.ecosystemType' : k;
-          if (!innerAcc[newKey]) {
-            innerAcc[newKey] = { partial: false, values: [] };
-          }
-          const value = isEcosystemTypes ? (d[k] as any[]).map((et) => et.ecosystemType) : d[k];
-          if (Array.isArray(value)) {
-            (innerAcc[newKey] as Record<string, any>).values.push(...value);
-          } else {
-            (innerAcc[newKey] as Record<string, any>).values.push(value);
-          }
-          return innerAcc;
-        }, acc);
-      }, {}) as FieldOptionsMap;
+      const result = (data ?? []) as FieldOptionsMap;
```

Comment:
`data` needs to be an object, it will not be undefined since the searchValues function returns null as a fallback.
This could be 
```
  const result = (data || {}) as FieldOptionsMap;
```